test/test_autograd.py

```diff
@@ -2040,6 +2040,21 @@ def run_test(input_size, exponent):
         run_test((10, 10), torch.zeros(10, 10))
         run_test((10,), 0)
 
+    def test_pinverse(self):
+        m, n = 5, 10
```

Comment:
Btw, can you add some comment here on why we are doing gradcheck in this way, i.e., constraints, input most be large enough and positive, etc.

Reply:
I have added it.